### DISCUSSION TITLE
Wow, I think it really is a great project, there are related to the deployment of the document?

### DISCUSSION THREAD

**Discussion by zkeq** [younho9/notion-url-shortener#2] [General]
Recently I am looking for a way to shorten the link .

Now I am through the cloud storage json file format to do it  :  [Tuostudy-Short-url](https://github.com/zkeq/Tuostudy-Short-url)

But I happened to see this project, wow, I want to say it's really cool . 🌟 

Could you please provide more deployment document about this project? I now for the `next.js` is not very familiar with .

And I want to know how to implement this project is to modify or delete a short link ? Or can implement a custom short link ?

Just like this ?  `http://tuo.icodeq.com/dream`  

I want to say again this project really look nice . ✨

> (Please forgive me may be some grammar mistakes, now I also in learning English too..  :)

**Comment by younho9** [comment]
Hello @zkeq ! Thank you for the feedback.

This project is a WIP. Maybe this weekend, I will work on writing some tests and document work. I will add a usage guide in the document.

This project is based on the [Notion official API](https://developers.notion.com/). Therefore, you will need to do prep works such as creating Notion account, getting the Notion API token, and duplicating Notion template page.

Thank you again for your interest and please wait for the document.

**Reply by zkeq** [reply]
🚀 Thanks.

**Reply by younho9** [reply]
You can now check some usage guide. It will be updated as development progresses. If you have difficulty following the guide or want to improve anything, please feedback. Thanks!

**Reply by zkeq** [reply]
Cool , I've been successfully deployed !

🚀 https://s.icodeq.com/thanks
